content/rc/security/vpc-peering.md

```diff
@@ -10,67 +10,124 @@ aliases:
 
 A [VPC peering connection](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html) is a networking connection between two virtual PCs that uses private IP addresses to route traffic between them.  Instances in either VPC can communicate with each other as if they are within the same network.
```

Comment:
@rrelledge this link points to AWS docs but it is part of the cloud agnostic section of this doc. Is that on purpose or an oversight?

Reply:
Good catch. I can take a look and see what other options might exist. A quick web search doesn't return any obvious agnostic resources for this though.